lib/decidim/file_authorization_handler/version.rb

```diff
@@ -2,11 +2,11 @@
 
 module Decidim
   module FileAuthorizationHandler
-    DECIDIM_VERSION = "0.29.2"
+    DECIDIM_VERSION = "0.30.1"
 
     # Uses the latest matching Decidim version for
     # - major, minor and patch
     # - the optional extra number is related to this module's patches
-    VERSION = "#{DECIDIM_VERSION}.0".freeze
+    VERSION = DECIDIM_VERSION.to_s.freeze
```

Comment:
Please, keep versioning semantic which is just explained in the comment
```suggestion
    VERSION = "#{DECIDIM_VERSION}.0".freeze
```